runtime/cudaq/utils/registry.h

```diff
@@ -7,8 +7,7 @@
  ******************************************************************************/
 
 #pragma once
-#define LLVM_DISABLE_ABI_BREAKING_CHECKS_ENFORCING 1
-#include "llvm/Support/Registry.h"
```

Comment:
Are you sure we should remove this? LLVM will crash if there is an ABI version mismatch.

Reply:
We don't use `llvm/Support/Registry.h` in this header. The other place (`runtime/common/Registry.h`) where we actually use `llvm::Registry` already has that `#define`.

We can compile without this `#include` here, so I guess it should be safe.

Reply:
This is old - from when the registry code was in utils and not in common. Safe to remove